src/effect/rounded_corners_effect.ts

```diff
@@ -62,12 +62,19 @@ export const RoundedCornersEffect = GObject.registerClass(
          * @param windowBounds - Bounds of the window without padding
          */
         updateUniforms(
+            focused: boolean,
             scaleFactor: number,
             config: RoundedCornerSettings,
             windowBounds: Bounds,
         ) {
             const borderWidth = getPref('border-width') * scaleFactor;
-            const borderColor = getPref('border-color');
+            const focusedBorderColor = getPref('focused-border-color');
+            const unfocusedBorderColor = getPref('unfocused-border-color');
+
+            var borderColor = focusedBorderColor;
+            if (!focused) {
+                borderColor = unfocusedBorderColor;
+            }
```

Comment:
@flexagoon what's a good way to rework this to just get the right color?  js isn't my forte.  Elsewhere i'd do something like `myColor = focused ? getPref('abc') : getPref('def')`

Reply:
> Elsewhere i'd do something like `myColor = focused ? getPref('abc') : getPref('def')`

You can do the same in JS

Reply:
Well shit, who knew ... that was easy enough.